packages/frontend/src/components/profile/ConfirmDisable/ui.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
```

Comment:
Maybe move everything under components/profile that is specific to profile page to
pages/profile ?
Similar with how we have pages/TokenSwap?

I'm dreaming of having components folder only contain dumb ui components

Reply:
It require additional refactoring. [Task created](https://mnw.atlassian.net/browse/MNW-257)